pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -2146,6 +2146,12 @@ var _ = Describe("Validating VM Admitter", func() {
 					Message: "Memory hotplug is only available for x86_64 and arm64 VMs",
 				}),
 				Entry("guest memory is less than 1Gi", func(vm *v1.VirtualMachine) {
+					vmArch := vm.Spec.Template.Spec.Architecture
+					//Skip the memory hotplug tests if the architecture is not amd64 or arm64
+					if vmArch != "amd64" && vmArch != "arm64" {
+						Skip(fmt.Sprintf("memory hotplug is not supported for the %s architecture.", vmArch))
```

Comment:
Can you instead skip the entire context when `rt.GOARCH == "s390x"`?

https://github.com/kubevirt/kubevirt/blob/a5701acf63fd1188846a2f7bfc230febbdb7356f/pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go#L2036-L2048

Or even better introduce a label we can skip when running on `s390x`.

https://onsi.github.io/ginkgo/#spec-labels

Reply:
sure

Reply:
done

Reply:
Hi @lyarwood  I have addressed your comments. Can you please review it again. Thanks